project.yaml

```diff
@@ -10,3 +10,12 @@ actions:
     outputs:
       highly_sensitive:
         dataset: output/dataset.csv.gz
+
+  generate_hyp_reg_dataset:
+      run: >
+        ehrql:v1
+        generate-dataset analysis/hyp_reg_dataset.py
+        --output output/hyp_reg/hyp017_viveck.csv.gz
```

Comment:
the code also looks correct, but I'm not sure why you named this `hyp017`? it looks like you implemented indicator `hyp001` (using the rules of `hyp_reg`)?

Reply:
Yup, I just checked. It should be hyp001, thank you!